nvflare/private/fed/app/deployer/server_deployer.py

```diff
@@ -84,6 +84,8 @@ def create_fl_server(self, args, secure_train=False):
             secure_train=secure_train,
             snapshot_persistor=self.snapshot_persistor,
             overseer_agent=self.overseer_agent,
+            shutdown_period=first_server.get("shutdown_period", 30.0),
+            check_engine_frequency=first_server.get("check_engine_frequency", 3.0),
```

Comment:
As Chester pointed out, we might want to make user to configure these 30.0, 3.0 number.

Reply:
Yes, this already enables the configurable from the config_fed_server.json